app/views/users/myaccount.html.erb

```diff
@@ -29,9 +29,9 @@
         <td><%= product.quantity%>
         <td><%= link_to "Edit Product", edit_product_path(product.id)%>
         <% if product.retired == false%>
-          <td><%= link_to "Retire", retire_path(product.id), method: :patch%>
+          <td><%= link_to "Retire", retire_path(product.id), method: 'retire'%>
```

Comment:
I believe the method needs to be an HTTP verb as a symbol like :post, :patch, :delete, etc.

Reply:
Idk why this wasn't working before, but I've changed all of the routes/method actions to patch and it still works! I'll commit those changes